sssd_test_framework/hosts/client.py

```diff
@@ -140,6 +140,7 @@ def backup(self) -> None:
             backup /etc/sssd "{location}/config"
             backup /var/log/sssd "{location}/logs"
             backup /var/lib/sss "{location}/lib"
+            backup /usr/libexec/sssd/passkey_child "{location}/passkey_child.orig"
```

Comment:
Is passkey_child always present or should the backup/restore check that the feature is enabled?

Reply:
now, we are not updating the client.py.